packages/field-base/test/active-mixin.test.js

```diff
@@ -0,0 +1,140 @@
+import { expect } from '@esm-bundle/chai';
+import {
+  arrowDownKeyDown,
+  enterKeyDown,
+  enterKeyUp,
+  fire,
+  fixtureSync,
+  isIOS,
+  mousedown,
+  mouseup,
+  spaceKeyDown,
+  spaceKeyUp,
+  touchend,
+  touchstart
+} from '@vaadin/testing-helpers';
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { ActiveMixin } from '../src/active-mixin.js';
+
+customElements.define(
+  'active-mixin-element',
+  class extends ActiveMixin(PolymerElement) {
+    static get template() {
+      return html`<div></div>`;
+    }
+  }
+);
+
+describe('active-mixin', () => {
+  let element;
+
+  beforeEach(() => {
+    element = fixtureSync(`<active-mixin-element></active-mixin-element>`);
+  });
+
+  describe('mouse', () => {
+    (isIOS ? it.skip : it)('should set active attribute on mousedown', () => {
```

Comment:
Let's update to use `isIOS ? describe.skip : describe` for the test suite and not individual tests.

Reply:
Done.